spec/features/send_message_spec.rb

```diff
@@ -1,17 +1,12 @@
 require 'rails_helper'
 
-describe 'Contact form' do
-  let(:user) { FactoryGirl.create(:user) }
-  before do
-    user.confirm!
-    user.update!(maintenance_mode: false)
-  end
+describe 'Contact form', :js do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.